src/compiler/program.ts

```diff
@@ -482,7 +480,7 @@ export function createCompilerHostWorker(
     const newLine = getNewLineCharacter(options);
     const realpath = system.realpath && ((path: string) => system.realpath!(path));
     const compilerHost: CompilerHost = {
-        getSourceFile: createGetSourceFile(fileName => compilerHost.readFile(fileName), () => options, setParentNodes, skipNonSemanticJSDocParsing),
+        getSourceFile: createGetSourceFile(fileName => compilerHost.readFile(fileName), () => options, setParentNodes),

```

Comment:
It's possible this isn't what you meant, @sheetalkamat; maybe you meant to make `createGetSourceFile` instead call the host, kinda like readFile?

Reply:
Your current change is what i meant in the commit comment before. So all good.

Reply:
Yay, thank you!!